src-internal/Admin/Schedulers/CustomersScheduler.php

```diff
@@ -3,13 +3,16 @@
  * Customer syncing related functions and actions.
  */
 
-namespace Automattic\WooCommerce\Admin\Schedulers;
+namespace Automattic\WooCommerce\Internal\Admin\Schedulers;
 
 defined( 'ABSPATH' ) || exit;
 
 use \Automattic\WooCommerce\Admin\API\Reports\Cache as ReportsCache;
 use \Automattic\WooCommerce\Admin\API\Reports\Customers\DataStore as CustomersDataStore;
-use \Automattic\WooCommerce\Admin\Schedulers\OrdersScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\ImportScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\OrdersScheduler;
+use Automattic\WooCommerce\Admin\Schedulers\WC_Order;
+use Automattic\WooCommerce\Admin\Schedulers\WP_User_Query;
```

Comment:
I might be lack understanding of this. I couldn't find the definition files for `Automattic\WooCommerce\Admin\Schedulers\WC_Order` and `Automattic\WooCommerce\Admin\Schedulers\WP_User_Query;`. Have we defined them somewhere?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/8316/commits/8d72e683f91470c16ef43fd47357051893983af2